src/theory/quantifiers/quantifiers_rewriter.cpp

```diff
@@ -1917,7 +1939,14 @@ Node QuantifiersRewriter::preSkolemizeQuantifiers( Node n, bool polarity, std::v
     Node nn = preSkolemizeQuantifiers( n[0], !polarity, fvTypes, fvs );
     return nn.negate();
   }else if( n.getKind()==kind::FORALL ){
-    if( polarity ){
+    if (n.getNumChildren() == 3)
```

Comment:
Since standard quantifiers can also have an attribute with triggers, change the comment below to reflect that.